build.gradle

```diff
@@ -21,12 +22,52 @@ reckon {
     stageFromProp('alpha', 'beta', 'rc', 'final')
 }
 
+facets {
```

Comment:
what's a `facet` ?! :thinking: 

quick google session does not point me to something that looks related ?

Reply:
https://github.com/nebula-plugins/nebula-project-plugin?tab=readme-ov-file#nebula-facet-plugin